themes/edsonarios/edsonarios.theme.sh

```diff
@@ -0,0 +1,19 @@
+#! bash oh-my-bash.module
+# Background: #333333
+# For unstaged(*) and staged(+) values next to branch name in __git_ps1
+GIT_PS1_SHOWDIRTYSTATE="enabled"
+OMB_PROMPT_VIRTUALENV_FORMAT=' [%s]'
+OMB_PROMPT_CONDAENV_FORMAT=' [%s]'
+
+STATUS_THEME_PROMPT_BAD="${_omb_prompt_bold_brown}❯_${_omb_prompt_normal} "
+STATUS_THEME_PROMPT_OK="${_omb_prompt_bold_green}❯_${_omb_prompt_normal} "
+
+function _omb_theme_PROMPT_COMMAND {
+  local ret_status="$( [ $? -eq 0 ] && echo -e "$STATUS_THEME_PROMPT_OK" || echo -e "$STATUS_THEME_PROMPT_BAD")"
+  local directory="$(if [[ ${PWD} == $HOME ]]; then echo '\W'; else echo ${PWD}; fi)/"
```

Comment:
Can you avoid using command substitutions, which involve forks? You can assign values inside the `if` statements. I'd appreciate it if you could also update the corresponding codes in the `purity` theme.

Reply:
Done